packages/ckeditor5-dev-web-crawler/src/runcrawler.ts

```diff
@@ -194,7 +194,7 @@ export default async function runCrawler( options: CrawlerOptions ): Promise<voi
 			message: ( error as Error ).message || '(empty message)'
 		} ) );
 
-		page.on( ERROR_TYPES.UNCAUGHT_EXCEPTION.event, error => onError( {
+		page.on( ERROR_TYPES.UNCAUGHT_EXCEPTION.event, error => pageErrors.push( {
```

Comment:
Am I right, the current production version is also affected?

https://github.com/ckeditor/ckeditor5-dev/blob/v47.1.1/packages/ckeditor5-dev-web-crawler/lib/runcrawler.js#L504-L508

If so, I would like to know if it wasn't done on purpose. Isn't it _uncaught_ meaning that it was unexpected?

@psmyrek, could you take a look?

Reply:
No, it was just that we had two implementation of `onError` - one that added the errors to the global / shared `errors` map and another that added to the local `pageErrors` array.

I changed it in #1108 to make the distinction clear.

Reply:
Hm. I think I got it.

https://github.com/ckeditor/ckeditor5-dev/blob/cde2cf9bf8754a3ad6be19849bb6b2b7c0200caa/packages/ckeditor5-dev-web-crawler/lib/runcrawler.js#L266-L271

Reply:
How should we test this PR?

Reply:
> @psmyrek, could you take a look?

This is a new implementation of error handling which differs from the first version of the crawler.